scalafix-tests/unit/src/test/scala/scalafix/tests/rule/SemanticTests.scala

```diff
@@ -40,7 +40,8 @@ object SemanticTests {
   )
   def classpath: Classpath = Classpath(
     List(
-      AbsolutePath(BuildInfo.semanticSbtClasspath),
+      // TODO: Figure out whether this is necessary.
+      // AbsolutePath(BuildInfo.semanticSbtClasspath),
```

Comment:
If `semanticdb-sbt` is still relevant, we need to merge and publish https://github.com/scalameta/semanticdb-sbt/pull/22 first.